docs/usage/openapi/ui_plugins.rst

```diff
@@ -0,0 +1,268 @@
+            :language: python
+
+    .. tab-item:: multiple
+
+        .. literalinclude:: /examples/openapi/plugins/serving_multiple_uis.py
+            :caption: Any combination of UIs can be served.
+            :language: python
+
+Configuring OpenAPI UI Plugins
+------------------------------
+
+Each plugin can be tailored to meet your unique requirements by passing options at instantiation. For full details on
+each plugin's options, see the :doc:`API Reference </reference/openapi/plugins>`.
+
+All plugins support the following options:
+
+- ``path``: Each plugin has its own default, e.g., ``/rapidoc`` for RapiDoc. This can be overridden to serve the UI at
+  a different path.
+- ``media_type``: The default media type for the plugin, typically ``text/html``.
+- ``favicon``: A string that should be a valid ``<link>`` tag, e.g.,
+  ``<link rel="icon" href="https://example.com/favicon.ico">``.
+- ``style``: Default is ``body { margin: 0; padding: 0; }``. This is applied to the ``<style>`` tag in the HTML rendered
+  by the plugin.
+
+Most plugins support the following additional options:
+
+- ``version``: The version of the UIs JS and (in some cases) CSS bundle to use. We use the ``version`` to construct the
+  URL to retrieve the the bundle from ``unpkg``, e.g., ``https://unpkg.com/rapidoc@<version>/dist/rapidoc-min.js``
+- ``js_url``: The URL to the JS bundle. If provided, this will override the ``version`` option.
+- ``css_url``: The URL to the CSS bundle. If provided, this will override the ``version`` option.
+
+Here's some examples of configuring the plugins:
+
+.. tab-set::
+
+    .. tab-item:: scalar
+
+        .. literalinclude:: /examples/openapi/plugins/scalar_config.py
+            :language: python
+
+    .. tab-item:: rapidoc
+
+        .. literalinclude:: /examples/openapi/plugins/rapidoc_config.py
+            :language: python
+
+    .. tab-item:: redoc
+
+        .. literalinclude:: /examples/openapi/plugins/redoc_config.py
+            :language: python
+
+    .. tab-item:: stoplight
+
+        .. literalinclude:: /examples/openapi/plugins/stoplight_config.py
+            :language: python
+
+    .. tab-item:: swagger
+
+        .. literalinclude:: /examples/openapi/plugins/swagger_ui_config.py
+            :language: python
+
+Configuring the OpenAPI Root Path
+---------------------------------
+
+The OpenAPI root path is the path at which the OpenAPI representations are served. By default, this is ``/schema``.
+This can be changed by setting the :attr:`OpenAPIConfig.path` attribute.
+
+In the following example, we configure the OpenAPI root path to be ``/docs``:
+
+.. literalinclude:: /examples/openapi/customize_path.py
+    :language: python
+
+This will result in any of the OpenAPI endpoints being served at ``/docs`` instead of ``/schema``, e.g.,
+``/docs/openapi.json``.
+
+Backward Compatibility
+----------------------
+
+OpenAPI UI plugins are a new feature introduced in ``v2.5.0``.
```

Comment:
```suggestion
OpenAPI UI plugins are a new feature introduced in ``v2.5.0``.
```

Change version here (?) but also should just use the [`..versionadded: vX.Y.Z` directive](https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-versionadded)

Reply:
Maybe at the top of the docs, or in the docstring for the plugins, but as this is a section about backward compatibility it serves as an introduction to the section IMO.

Reply:
I have added it at the top of the page.